src/Sarif.FunctionalTests/DirectProducerTestData/WebSkim/HtmlAttributeAnalysisFixes.sarif

```diff
@@ -0,0 +1,288 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-1.0.0",
+  "version": "1.0.0",
+  "runs": [
+    {
+      "id": "de711026-6d1c-4760-a2ca-c0d94b93312b",
+      "tool": {
+        "name": "ModernCop",
+        "fullName": "mod50 7.0.7000.0",
+        "version": "7.0.7000.0",
+        "semanticVersion": "7.0.7000",
+        "fileVersion": "7.0.20811.0",
+        "language": "en-US",
+        "properties": {
+          "CompanyName": "Microsoft Corporation"
+        }
+      },
+      "results": [
+        {
+          "ruleKey": "WEB1067.AttributeNameIsNotLowercase",
+          "level": "error",
+          "formattedRuleMessage": {
+            "formatId": "Default",
+            "arguments": [
+              "ALT",
+              "area"
+            ]
+          },
+          "locations": [
+            {
+              "analysisTarget": {
+                "uri": "file:///d:/src/SecDevTools/Main/src/ModernCop/Source/WebRules.Test.Unit/ModernCopTestData/WebRules_HtmlRules/Attributes/AttributeAnalysis.html",
+                "region": {
+                  "startLine": 17,
+                  "startColumn": 54,
+                  "endColumn": 57,
+                  "offset": 561,
+                  "length": 3
+                }
+              }
+            }
+          ],
+          "fixes": [
+            {
+              "description": null,
+              "fileChanges": [
+                {
+                  "uri": null,
+                  "replacements": [
+                    {
+                      "offset": 561,
+                      "deletedLength": 3,
+                      "insertedBytes": "YWx0"
+                    }
+                  ]
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "ruleKey": "WEB1079.AttributeValueIsNotQuoted",
+          "level": "error",
+          "formattedRuleMessage": {
+            "formatId": "Default",
+            "arguments": [
+              "shape"
+            ]
+          },
+          "locations": [
+            {
+              "analysisTarget": {
+                "uri": "file:///d:/src/SecDevTools/Main/src/ModernCop/Source/WebRules.Test.Unit/ModernCopTestData/WebRules_HtmlRules/Attributes/AttributeAnalysis.html",
+                "region": {
+                  "startLine": 18,
+                  "startColumn": 105,
+                  "endColumn": 110,
+                  "offset": 743,
+                  "length": 5
+                }
+              }
+            }
+          ],
+          "fixes": [
+            {
+              "description": "Wrap attribute value in single quotes.",
+              "fileChanges": [
+                {
+                  "uri": null,
```

Comment:
This isn't right. The [`fileChange.uri`](https://rawgit.com/sarif-standard/sarif-spec/master/Static%20Analysis%20Results%20Interchange%20Format%20(SARIF).html#filechange-uri) property is required (and even if it weren't, we shouldn't be emitting a default value).